### ISSUE TITLE
CPFP integration: a "priority" toggle before setting a Spend transaction

### ISSUE THREAD

**Issue by darosior** [revault/revault-gui#313]
`revaultd` now allows for setting a Spend transaction as "High Priority" in `setspendtx`. It will then feebump it if its feerate is lower than the next block estimated feerate at the time of broadcast.

We should allow the user to set such a transaction as "high prio" on the GUI. I think a toggle just before the "broadcast" button makes sense but it's of course up for debate.

**Comment by darosior** [comment]
Some more points brought up by Daniela on IRC:
```
11:34 <danielabrozzoni> About CPFP, I'd say:
11:34 <danielabrozzoni> - Priority button when broadcasting, maybe with a (?) button somewhere to explain what happens if you set priority
11:34 <danielabrozzoni> - Some popup or something that tells you not to close the GUI after broadcasting if you chose priority
```

**Comment by darosior** [comment]
Let's go with the priority toggle first, and we'll open an issue for the popup as an enhancement of this feature.

**Comment by edouardparis** [comment]
closed by #328
